experiments/dutch_auctions/abaci.py

```diff
@@ -0,0 +1,69 @@
+""" Abaci Module
+    Class-based representation of the Abaci smart contract
+    (contains only what is necessary for the simulation)
+"""
+
+from pydss.pymaker.numeric import Wad, Rad
+from pydss.util import require
+
+
+class Abaci:
+    """
+    """
+
+    def __init__(self):
+        pass
+
+    def file(self, what, data):
+        pass
+
+    def price(self, top, dur):
+        """
+        top: initial price
+        dur: seconds since action has started
+        """
+        pass
+
+
+class LinearDecrease(Abaci):
+    """
+    tau = int
+    """
+
+    def __init__(self):
+        self.tau = 0
+        super().__init__()
```

Comment:
don't worry about the `super().__init__()` call here, since in `super()` the `__init__()` function is basically undefined